### ISSUE TITLE
Support for 3.11.2 ?

### ISSUE THREAD

**Issue by hkroger** [Stratio/cassandra-lucene-index#382]
Hi, 

3.11.2 is out. Do you have expected schedule for update ?

**Comment by kotdv** [comment]
I couldn't wait, so I forked it and updated to support 3.11.2. 
You can pull it and build the plugin from it until there's an "official bump": https://github.com/kotdv/cassandra-lucene-index/tree/branch-3.11.1

You can also compare and see what I changed (just in case): 
https://github.com/Stratio/cassandra-lucene-index/compare/branch-3.11.1...kotdv:branch-3.11.1

**Comment by hkroger** [comment]
In other words the old plugin would work just fine with new version if this works?

**Comment by kotdv** [comment]
Yeah, I've checked the differences between several previous releases and I haven't found anything worth changing, considering this is a minor version update for Cassandra itself.

**Comment by hkroger** [comment]
from 3.11.0 => 3.11.1 there were some incompatible changes and lucene didn't compile without actually changing some code. So this time we had more luck :)

**Comment by kotdv** [comment]
I have it deployed on kubernetes, aaand so far so good.

**Comment by roadrunner** [comment]
@kotdv can you share the docker file and deployment?

**Comment by akshitjain** [comment]
@stratiocommit @stratioadmin Is there any timeline by which 3.11.2 startio-lucene will be released?

**Comment by hkroger** [comment]
@akshitjain we just deployed 3.11.2 with plugin for 3.11.1. Works fine.

**Comment by engin** [comment]
@hkroger i can confirm that as well

https://hub.docker.com/r/dumlu/cassandra-lucene-docker/
